index.js

```diff
@@ -59,6 +59,7 @@ client.once('ready', () => {
   });
   const Guilds = client.guilds.cache.map(guild => guild.name);
   console.log(Guilds, 'Connected!');
+  Canvas.registerFont('./resources/welcome/OpenSans-Light.ttf', { family: 'Open Sans Light' }); // Registering font For Cloud Services
 });
```

Comment:
Heads up it was added because if I restarted the bot it would sometime trigger warning that may confuse the user  could trigger an Error